src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -4071,6 +4222,7 @@ static const StreamListItem streams_list[] = {
 	StreamListItem(&MavlinkStreamGlobalPositionInt::new_instance, &MavlinkStreamGlobalPositionInt::get_name_static, &MavlinkStreamGlobalPositionInt::get_id_static),
 	StreamListItem(&MavlinkStreamLocalPositionNED::new_instance, &MavlinkStreamLocalPositionNED::get_name_static, &MavlinkStreamLocalPositionNED::get_id_static),
 	StreamListItem(&MavlinkStreamVisionPositionEstimate::new_instance, &MavlinkStreamVisionPositionEstimate::get_name_static, &MavlinkStreamVisionPositionEstimate::get_id_static),
+	StreamListItem(&MavlinkStreamOdometry::new_instance, &MavlinkStreamOdometry::get_name_static, &MavlinkStreamOdometry::get_id_static),
```

Comment:
This loopback is not useful, as I said above.

Reply:
As I said, I think it makes sense to have only a msg to parse the odometry data. One of the reasons is the sync, as you also said, which the OBC also requires to do.

Reply:
Synchronising topics using timestamps is trivial on the companion side. On the autopilot side due to buffering/memory requirements, it is not as easy. I still suggest getting rid of the ODOMETRY loopback.